cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -494,28 +472,57 @@ def gen_json_out(self):
                 }
             }
         ]
+
+        # if 'CSI_RBD_PROVISIONER_SECRET' exists, then only add 'rook-csi-rbd-provisioner' Secret
+        if self.out_map['CSI_RBD_PROVISIONER_SECRET']:
+            json_out.append({
+                "name": "rook-csi-rbd-provisioner",
+                "kind": "Secret",
+                "data": {
+                    "userID": 'csi-rbd-provisioner',
+                    "userKey": self.out_map['CSI_RBD_PROVISIONER_SECRET']
+                },
+            })
+        # if 'CSI_CEPHFS_PROVISIONER_SECRET' exists, then only add 'rook-csi-cephfs-provisioner' Secret
+        if self.out_map['CSI_CEPHFS_PROVISIONER_SECRET']:
+            json_out.append({
+                "name": "rook-csi-cephfs-provisioner",
+                "kind": "Secret",
+                "data": {
+                    "adminID": 'csi-cephfs-provisioner',
+                    "adminKey": self.out_map['CSI_CEPHFS_PROVISIONER_SECRET']
+                },
+            })
+        # if 'CSI_CEPHFS_NODE_SECRET' exists, then only add 'rook-csi-cephfs-node' Secret
+        if self.out_map['CSI_CEPHFS_NODE_SECRET']:
+            json_out.append({
```

Comment:
what special with rbd node plugin secret its need to be handled in a different way?